go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/microsoft/ethr
 
 require (
-	github.com/mattn/go-runewidth v0.0.3
-	github.com/nsf/termbox-go v0.0.0-20181027232701-60ab7e3d12ed
-	github.com/pkg/errors v0.8.1
-	golang.org/x/sys v0.0.0-20181206074257-70b957f3b65e
+	github.com/mattn/go-runewidth v0.0.9
+	github.com/nsf/termbox-go v0.0.0-20200418040025-38ba6e5628f1
+	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/common v0.10.0
+	golang.org/x/sys v0.0.0-20200918174421-af09f7315aff
```

Comment:
Are these version bumps required for the Prometheus exporter tool to work? Otherwise this feels unrelated to the purpose of the PR. If these are harmless and/or beneficial despite not being related to the Prometheus exporter maybe let's have them on a separate PR?